### DISCUSSION TITLE
@Urgent!!! AdGuard Disabling Essential Banking Application - Immediate Rectification Required

### DISCUSSION THREAD

**Discussion by jameschung0225** [List-KR/community#17] [General]
Urgent!!!
AdGuard is currently causing an egregious disruption to a key banking application in Korea, making it entirely non-operational. This is due to your service blocking the DNS resolution of mbp.hanabank.com, a critical component of the application. 

This level of service disruption is utterly unacceptable and requires immediate and decisive action. I am demanding immediate rectification, specifically the restoration of the DNS resolution for mbp.hanabank.com, and the subsequent re-establishment

**Comment by piquark6046** [comment]
I heard that an employee of hanabnak's zendesk ticket related to the issue was submitted and is being processed. During processing time, please use Cloudflare DNS server instead of AdGuard DNS server temporary. 
@ameshkov will handle the issue.

**Comment by jameschung0225** [comment]
AdGuard is currently causing an egregious disruption to a key banking application in Korea, making it entirely non-operational. This is due to your service blocking the DNS resolution of mbp.hanabank.com, a critical component of the application. 

This level of service disruption is utterly unacceptable and requires immediate and decisive action. I am demanding immediate rectification, specifically the restoration of the DNS resolution for mbp.hanabank.com, and the subsequent re-establishment of the banking application's functionality. Any further delays will not be tolerated. 

Your immediate and undivided attention to this matter is not a request, but a requirement.

**Reply by piquark6046** [reply]
Actually, I have not the permission related to manage AdGuard DNS server.
Please wait until `@ameshkov` handles it.

**Reply by piquark6046** [reply]
Please refer https://github.com/List-KR/List-KR/discussions/732#discussioncomment-6292291.
